code/web/src/modules/survey/api/actions.js

```diff
@@ -0,0 +1,19 @@
+import axios from 'axios';
+import { mutation } from 'gql-query-builder';
+import { routeApi } from '../../../setup/routes';
+
+export const SET_USER_STYLE = 'AUTH/SET_USER_STYLE';
+
+export const postUserStyle = (email, userStyle) => {
+  return axios.post(
```

Comment:
Really hard won figuring out this syntax for this post request to the backend `userStyleUpdate` query! Well done team.